internal/configs/virtualserver.go

```diff
@@ -2056,7 +2069,7 @@ func createUpstreamsForPlus(
 
 	isPlus := true
 	upstreamNamer := newUpstreamNamerForVirtualServer(virtualServerEx.VirtualServer)
-	vsc := newVirtualServerConfigurator(baseCfgParams, isPlus, false, staticParams)
+	vsc := newVirtualServerConfigurator(baseCfgParams, isPlus, false, staticParams, false)
```

Comment:
Hardcoded to `false` here under the assumption that wildcard cert support does not affect upstream generation, but I am not really familiar with the codebase.